src/processors/models/processorsManager.js

```diff
@@ -68,6 +68,23 @@ module.exports.redownloadJSProcessor = async function(processorId) {
     }
 };
 
+module.exports.updateProcessor = async function (processorId, processor) {
+    const oldProcessor = await databaseConnector.getProcessor(processorId);
+    if (!oldProcessor) {
+        throw generateProcessorNotFoundError();
+    }
+    processor.created_at = oldProcessor.created_at;
+    if (processor.type === 'file_download') {
```

Comment:
lets change to 

` if (processor.type === constants.PROCESSOR_TYPE_FILE_DOWNLOAD && processor.file_url !==  oldProcessor.file_url) {`

Reply:
as we discussed, let's change to 

`if (processor.type === constants.PROCESSOR_TYPE_FILE_DOWNLOAD && req.query.dl) {`

Reply:
done